articles/openshift/oshift-how-add-domains-proxy-whitelist.md

```diff
@@ -0,0 +1,122 @@
+---
+title: How to permit outbound access to internet hosts in deployments with public services network connectivity | UKCloud Ltd
```

Comment:
By public services network do you mean the PSN or more generally government community networks?